node/src/accountant/db_access_objects/receivable_dao.rs

```diff
@@ -363,27 +438,29 @@ impl ReceivableDaoReal {
         )
     }
 
-    fn more_money_received_pretty_error_log(
-        &self,
+    fn more_money_received_roll_back_error_log(
```

Comment:
If I were writing this code, I would've migrated this function into a separate file and I've kept the nested functions outside this function. I find that way easier to read, but it's up to you if you want to do that.